documentation/slate/source/includes/apis.md

```diff
@@ -935,6 +936,43 @@ The [isochrones](#isochrones) service exposes another response structure, which
 | nop     | wheelchair           | boolean | If true the traveler is considered to be using a wheelchair, thus only accessible public transport are used<br>be warned: many data are currently too faint to provide acceptable answers with this parameter on       | False       |
 | nop     | debug                | boolean | Debug mode<br>No journeys are filtered in this mode     | False       |
 
+### Precisions on `forbidden_uris[]` and `allowed_id[]`
+
+These parameters are filtering the vehicle journeys and the stop points use to compute the journeys.  The journeys can only use allowed vehicle journeys (as present in the `public_transport` or `on_demand_transport` sections). They also can only use the allowed stop points (as present in the `street_network`, `waiting` and `crow_fly` sections).
```

Comment:
:+1: seems quite clear for me.

Hovewer I don't find the hand drawn picture cristal clear, I think a computer generated schema would be better